Doc/data/stable_abi.dat

```diff
@@ -286,9 +286,6 @@ PyFrame_GetLineNumber
 PyFrozenSet_New
 PyFrozenSet_Type
 PyGC_Collect
-PyGC_Disable
```

Comment:
This doesn't look right to me and don't know how to read this. 

This was the output of `make regen-limited-abi`

Reply:
@encukou - Please verify if this is correct.

Reply:
No, that's not correct. Those should stay in the file.

Reply:
They probably need to get added to `Misc/stable_abi.txt` now. That seems to be the new source of truth. I'll add them there.

Reply:
@scoder - Thank you. I could update this PR with Misc/stable_abi.txt change too.

Reply:
Petr took care of it - https://github.com/python/cpython/pull/25720